tests/regression/nextcloud-rule-exclusions-plugin/9508993.yaml

```diff
@@ -0,0 +1,26 @@
+---
+meta:
+  author: "Jean-Kevin Kpadey"
+  description: "Nextcloud Rule Exclusions Plugin"
+  enabled: true
+  name: 9508993.yaml
+tests:
+  - test_title: 9508993-1
+    desc: |
+      Moving cards on Deck apps.
+      Target: description
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              Host: localhost
+              User-Agent: OWASP ModSecurity Core Rule Set
+              Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
+            port: 80
+            method: PUT
+            uri: /apps/deck/cards/1/reorder
+            data: "description=This is a description.\n\nUse this [link](https://github.com/microsoft/vscode/issues?page=3&q=is%3Aissue+is%3Aopen).\n"
+          output:
+            no_log_contains: |
+              id"911100"|id "93(?:[23]2[10]0)"|id "94(?:2[234][03467]0|9110)"|id "980130"
```

Comment:
I appreciate the optimisation of the regular expression, however, in this situation readability is much more important. Please write use full rule numbers; using a single expression to match multiple rules with the last digit is fine IMO, e.g., `93223[156]`. There's also a missing space after the first `id`.

Reply:
I listed all ids